sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/MapElements.java

```diff
@@ -25,7 +25,7 @@
  * {@code PTransform}s for mapping a simple function over the elements of a {@link PCollection}.
  */
 public class MapElements<InputT, OutputT>
-extends PTransform<PCollection<InputT>, PCollection<OutputT>> {
+extends PTransform<PCollection<? extends InputT>, PCollection<OutputT>> {
```

Comment:
This particular change was not. There were cases that could not be ported without it, since the new generic makes it the same type as `ParDo`.